pages/index.vue

```diff
@@ -1,41 +1,50 @@
 <template>
-  <!-- Buttons -->
-  <div class="flex flex-col place-content-evenly h-full p-24">
-    <div class="flex gap-48 place-content-center h-full">
-      <div class="border-4 rounded-lg w-1/3 min-h-2/3 p-8">
+  <div class="flex flex-col p-24">
```

Comment:
Does reinstating the h-full here fix the scrollbar problem?

Reply:
Oh I think it's the `w-full` on the images.

Reply:
Can we look at this in ± an hour with screenshare? I don't think I fully understand the issue so I'm not sure how to fix.